ui/src/app/tasks/task-schedule-create/task-schedule-create.component.html

```diff
@@ -0,0 +1,191 @@
+    <div *ngIf="schedule.taskDefinitions.length == 1">
+      <div class="row" style="margin-bottom: 12px">
+        <div class="col-xs-4">
+          <label>Task Name:</label>
+        </div>
+        <div class="col-xs-20">
+          <a style="cursor: pointer" (click)="detailsTask(schedule.taskDefinitions[0])">
+            {{ schedule.taskDefinitions[0] }}</a>
+        </div>
+      </div>
+      <div class="row form-group" [class.has-error]="form.get('names').controls[0].invalid">
+        <div class="col-xs-4">
+          <label class="control-label" for="name">Scheduler name:</label>
+        </div>
+        <div class="col-xs-20">
+          <div formArrayName="names">
+            <input id="name" name="name" [formControl]="form.get('names').controls[0]" class="form-control input-sm"/>
+            <div class="errors-name" *ngIf="form.get('names.0').invalid">
+              <div *ngIf="form.get('names.0').errors?.required" class="help-block">The name is required.</div>
+              <div *ngIf="form.get('names.0').errors?.exist" class="help-block">The name already exists.</div>
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
+    <div *ngIf="schedule.taskDefinitions.length > 1">
+      <div class="row form-group">
+        <div class="col-xs-4">
+          <label class="control-label">Tasks:</label>
+        </div>
+        <div class="col-xs-20">
+          <div class="alert alert-danger" style="max-width: 700px" *ngIf="form.get('names').errors?.notUnique">
+            Duplicate schedule name on the form.
+          </div>
+          <table class="table table-form table-form-stripped" style="max-width: 700px">
+            <thead>
+            <tr>
+              <th style="padding-top: 4px" width="50%">Task</th>
+              <th style="padding-top: 4px" width="50%">Schedule name</th>
+            </tr>
+            </thead>
+            <tbody>
+            <tr *ngFor="let item of schedule.taskDefinitions; index as i">
+              <td class="td-label"><a style="cursor: pointer" (click)="detailsTask(item)">{{ item }}</a></td>
+              <td>
+                <div [class.has-error]="form.get('names.' + i).invalid">
+                  <input [formControl]="form.get('names').controls[i]" type="text" class="form-control input-sm"/>
+                  <div *ngIf="form.get('names.' + i).invalid">
+                    <div *ngIf="form.get('names.' + i).errors?.required" class="help-block">The name is required.</div>
+                    <div *ngIf="form.get('names.' + i).errors?.exist" class="help-block">The name already exists.</div>
+                  </div>
+                </div>
+              </td>
+            </tr>
+            </tbody>
+          </table>
+        </div>
+      </div>
+    </div>
+    <div class="row form-group" [class.has-error]="form.get('cron').invalid">
+      <div class="col-xs-4">
+        <label class="control-label" for="cron">Cron Expression:</label>
```

Comment:
We should provide some usage examples for users. Who remembers the CRON syntax? ;-)

Reply:
Also maybe detail the syntax according to PCF docs.